docs/ia.md

```
@@ -97,10 +97,10 @@
 └── [노쇼 처리]
 ```
 
-## 3.5 관리자 - 가게 승인 플로우
+## 3.5 관리자 - 판매자 승인 플로우
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)